README.md

```diff
@@ -1,6 +1,128 @@
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://onjoseph.github.io/metrics-webapp/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Internet connection and browser
+- A source code editor(preferably Visual Studio Code)
+- Browser
+- Install nodeJS along with NPM
+- Interactions
+  - Home page
+    - When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.
+    - Along with the category name, you will display some numeric values per category that come from the API.
+    - When the user clicks (or taps) on a category item, the application navigates to the details page.
+
+  - Details page
+    - In the details page, the webapp retrieves data from the API to show detailed data in the category.
+    - When the user clicks on the "Back" button (<), the user navigates to the home page.
+
+- Testing requirements
+  - Create unit tests for pure functions in your Redux code.
+  - Create integration tests for your application using the React Testing Library.
+    - You may need to mock the access to the API, so that your tests don't send actual requests.
+    - You may need to mock the connection to the Redux Store.
+
+- Technical requirements
+  - The project is a single-page application (SPA) built with React and Redux.
+  - The data retrieved from the API should be stored in the Redux store.
+  - You should filter the data that you retrieve from the API using a Filter stateless component.
+  - Every page (the main page and the pages for each item) should have a unique route within the SPA.
+  - The project should be deployed and accessible online (add a link to the online version of your app to your README file.
+
+Setting up this project locally:
+#### On your terminal:
+- Clone this repository to your computer: https://github.com/ONJoseph/metrics-webapp.git
+- Open the project on your code editor: "cd metrics-webapp" and "code ."
+- Install npm package: npm install.
+- Run website: npm run start.
+
+-Local environment with `https://github.com/ONJoseph/metrics-webapp.git`
+
+- This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+### Install
+
+Install this project with:
+
+- [Git](https://git-scm.com/downloads)
+- [Node](https://nodejs.org/en/download/)
+- Run `npm i` on the root of the project and you're good to go. All of the necessary packages will be installed for you!
+
+### Usage
+
+To run the project, execute the following command:
+
+- Clone the repository using `https://github.com/ONJoseph/metrics-webapp.git`
+- Change directory into the project folder: `cd metrics-webapp`
+- Run `npm install`
+- Run `npm start`
+- A new browser will open automatically with application loaded
```

Comment:
- [ ] I have double checked in both your README and the pull request summary and I have not seen the link to the project video demo. Kindly take it and include it in your project.

![image](https://user-images.githubusercontent.com/86318284/219403456-13c45326-54e2-4b98-bd01-419b039bf05f.png)

Reply:
Hi @Nemwel-Boniface , Thanks a lot for the review! I have added the link to the project documentation video. Best regards!